tests/api.c

```diff
@@ -71980,15 +71980,12 @@ static int test_wolfSSL_SESSION_expire_downgrade(void)
     !defined(NO_RSA) && defined(HAVE_IO_TESTS_DEPENDENCIES) && \
     !defined(NO_SESSION_CACHE) && defined(OPENSSL_EXTRA) && \
     !defined(WOLFSSL_NO_TLS12)
-
-    WOLFSSL_CTX* ctx = NULL;
     callback_functions server_cbf, client_cbf;
 
     XMEMSET(&server_cbf, 0, sizeof(callback_functions));
     XMEMSET(&client_cbf, 0, sizeof(callback_functions));
 
     /* force server side to use TLS 1.2 */
-    server_cbf.ctx = ctx;
     server_cbf.method = wolfTLSv1_2_server_method;
 
```

Comment:
In `test_wolfSSL_SESSION_expire_downgrade` we are trying to test that we correctly reject an expired session. Now that the `ctx` is being changed we will always reject the session since it doesn't match with the `ctx`. Are you sure we are still testing the same thing?

Reply:
I just checked and we do hit the line `Invalid session: timed out` so it looks like we are good. I don't remember then why I used a shared context here.